### PR TITLE
docs: Add tagged regions to changelog

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Adds list of links and two tagged regions to the changelog to use with https://github.com/elastic/observability-docs/pull/3476.

### CODE REVIEW

**Review by bmorelli25 — APPROVED** [review]
Love that everything related to release notes will now live in one directory.

APM Server team, please merge this after you approve! 🙇

**Review by axw — APPROVED** [review]

**Pull request merged**
